### ISSUE TITLE
add alt text to all images and figures in .rst pages

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc-doc#20]

**Comment by DrXyzzy** [comment]
Current alt text coverage in doc pages is 154 out of 358 images and figures, or 43%:

```
~/cocalc-doc/source$ git grep --extended-regexp "(image|figure)::" | wc -l
358
~/cocalc-doc/source$ git grep "alt:" | wc -l
154
~/cocalc-doc/source$ echo $((154*100/358))%
43%
```

**Comment by DrXyzzy** [comment]
Current alt text coverage in doc pages is 203 out of 362 images and figures, or 56%:

**Comment by DrXyzzy** [comment]
Current alt text coverage in doc pages is 247 out of 393 images and figures, or 62%:

**Comment by DrXyzzy** [comment]
Current alt text coverage in doc pages is 346 out of 464 images and figures, or 74%.

**Comment by DrXyzzy** [comment]
Current alt text coverage in doc pages is 377 out of 491 images and figures, or 76%.

**Comment by DrXyzzy** [comment]
The following 21 files in cocalc-doc may have figure or image objects with missing alt text. In addition to adding alt text, these articles should be reviewed because they are probably outdated:

- [ ] account/support.rst
- [ ] howto/activeloop-hub.rst
- [ ] howto/dask.rst
- [ ] howto/export-timetravel.rst
- [ ] howto/fortran.rst
- [ ] howto/google-api.rst
- [ ] howto/gsl.rst
- [ ] howto/mongodb.rst
- [ ] howto/neuron.rst
- [ ] howto/overleaf.rst
- [ ] howto/plotly-dash.rst
- [ ] howto/project-invitation-tokens.rst
- [ ] howto/ruby.rst
- [ ] howto/shiny-server.rst
- [ ] howto/upload.rst
- [ ] latex-faq.rst
- [ ] latex-features.rst
- [ ] platform-news.rst
- [ ] snippets.rst
- [ ] software.rst
- [ ] teaching-students.rst

Script to screen for missing alt tags
```bash
#!/bin/bash

# usage: cmd [alt|fig]

cd ~/cocalc-doc/source

case $1 in
  alt)
git ls-files |
  grep ".rst$" |
  xargs -d '\n' grep -l ":alt:" |
  sort |
  less
  ;;

  fig)
git ls-files |
  grep ".rst$" |
  xargs -d '\n' egrep -l "(image|figure)::" |
  sort |
  less
  ;;

  *)
  echo ERROR
  exit 1
esac
```

**Comment by DrXyzzy** [comment]
Done.
